napari/layers/labels/_labels_mouse_bindings.py

```diff
@@ -0,0 +1,37 @@
+from ._labels_utils import interpolate_coordinates
+
+
+def paint(layer, event):
+    """Paint with the currently selected label."""
+    # on press
+    layer._save_history()
+    layer._block_saving = True
```

Comment:
This looks like it should be a context manager...

Reply:
Agreed, that should be a follow-up PR, issue made #1065